### ISSUE TITLE
Conversion Issue

### ISSUE THREAD

**Issue by chidiewenike** [MycroftAI/mycroft-precise#17]
For converting the Public Domain Sounds Backup dataset to wav, I keep getting this error: 

"Output file #0 does not contain any stream."

Any idea why?

**Comment by MatthewScholefield** [comment]
What command are you using?

**Comment by chidiewenike** [comment]
"SOURCE_DIR=data/random/mp3
DEST_DIR=data/random

for i in $SOURCE_DIR/*.mp3; do echo "Converting $i..."; fn=${i##*/}; ffmpeg "$i" -acodec pcm_s16le -ar 16000 -ac 1 "$DEST_DIR/${fn%.*}"; done"

I followed the steps provided in the wiki but I keep getting this error.

**Comment by MatthewScholefield** [comment]
@chidiewenike I think I forgot the `-i` flag to `ffmpeg`. I've updated the wiki. Let me now if this works:
```bash
SOURCE_DIR=data/random/mp3
DEST_DIR=data/random

for i in $SOURCE_DIR/*.mp3; do echo "Converting $i..."; fn=${i##*/}; ffmpeg -i "$i" -acodec pcm_s16le -ar 16000 -ac 1 "$DEST_DIR/${fn%.*}"; done
```

**Comment by chidiewenike** [comment]
That fixed the ""Output file #0 does not contain any stream." issue. It did indicate that there was no output format provided so I added "-f wav" to provide the output format and it works now. 

Thank you

**Comment by MatthewScholefield** [comment]
Thanks, I've added that to the wiki as well.
